extensions/amp-analytics/0.1/vendors.js

```diff
@@ -76,6 +76,7 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'viewer': 'VIEWER',
       'viewportHeight': 'VIEWPORT_HEIGHT',
       'viewportWidth': 'VIEWPORT_WIDTH',
+      'userAgent': 'USER_AGENT',
```

Comment:
The list is sorted by alphabetic order. Please follow and place `userAgent` to the correct place